test/unit/services/publish_finder_test.rb

```diff
@@ -1,26 +1,29 @@
 require 'test_helper'
 
 class PublishFinderTest < ActiveSupport::TestCase
```

Comment:
Not sure if you need to change this test - it's not testing anything new with if we're doing this multiple times.